src/content/recipes/pinia.md

```diff
@@ -0,0 +1,85 @@
+<div class="aside aside__no-top">
+
+This recipe assumes that you are using Vue 3 and Storybook >= 7.x. If you haven't upgraded yet, run the following command:
+
+```shell
+npx storybook@latest upgrade
+```
+
+</div>
+
+<RecipeHeader>
+
+How to setup Pinia and Storybook
+
+</RecipeHeader>
+
+Pinia is the recommended standard for state management in Vue. It gives you a lightweight and type-safe way to handle global state for your applications. This recipe will show you how to integrate Pinia into Storybook so that you can test your state connected components.
+
+If you want to look at a full example, look at this [awesome repo](https://github.com/chakAs3/vue3-pinia-storybook/tree/main) built by [ChakAs3](https://github.com/chakAs3).
+
+## Initialize pinia
+
+Inside of `preview.ts`, import and initialize pinia.
+
+```ts
+import  { type Preview }  from '@storybook/vue3';
+
+import { createPinia } from 'pinia';
+
+const pinia = createPinia();
+
+
+const preview: Preview = {
```

Comment:
Is all this stuff needed?

Reply:
Yes, as we landed on using the fresh copy of preview.ts after init for context sake